### PR TITLE
Adds WPScan CVEs

### PR DESCRIPTION
**Pull request by erwanlr**

### CODE REVIEW

File: 2021/24xxx/CVE-2021-24964.json [hunk 1]
```diff
@@ -1,18 +1,76 @@
 {
```
Comment (cve-team) [inline 2021/24xxx/CVE-2021-24964.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/24xxx/CVE-2021-24964.json:1]: ("Expecting ',' delimiter: line 77 column 1 (char 2170)",)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/24xxx/CVE-2021-24964.json (hunk 1 above)
Comment (ktodd229) [reply 2021/24xxx/CVE-2021-24964.json:1]: Please check change requests and resubmit.

Comment (erwanlr) [reply 2021/24xxx/CVE-2021-24964.json:1]: This has been done already: https://github.com/CVEProject/cvelist/pull/3912/commits/70aabc675881c14c020059a4c0c63346db6bafdd

**Pull request merged**